tokio-util/src/task/mod.rs

```diff
@@ -11,3 +11,5 @@ pub use join_map::{JoinMap, JoinMapKeys};
 
 pub mod task_tracker;
 pub use task_tracker::TaskTracker;
+
+pub mod abort_on_drop;
```

Comment:
No reason to expose a module with one item.
```suggestion
mod abort_on_drop;
pub use abort_on_drop::AbortOnDropHandle;
```